content/en/docs/tasks/administer-cluster/encrypt-data.md

```diff
@@ -92,6 +92,13 @@ Name | Encryption | Strength | Speed | Key Length | Other Considerations
 Each provider supports multiple keys - the keys are tried in order for decryption, and if the provider
 is the first provider, the first key is used for encryption.
 
+__It is not recommended to use EncryptionConfig without using the `kms` provider, as this does not materially change your
+security posture.__ By default, the `identity` provider is used to protect secrets in etcd, which provides no encryption.
+`EncryptionConfiguration` was introduced in 1.7, to encrypt secrets locally, with a locally managed key. In this case, if etcd
+is compromised, there is no additional protection for secrets compared to when they are stored in plaintext. This was a
+stepping stone in development to the `kms` provider, introduced in 1.10. Envelope encryption creates dependence on a separate
```

Comment:
Where we mention the version of Kubernetes that something was introduced in, we've got the option of using a [feature-state shortcode](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#feature-state).

Does that feel like a good fit here?

Reply:
maybe? EncryptionConfig was introduced in 1.7 (presumably in alpha?) and is now GA, and kms provider was introduced in Alpha in 1.10, but is now in Beta. Part of what I'm trying to express is that just using EncryptionConfig on its own was not the intended end goal.
Would I then say, for EncryptionConfig, {{< feature-state feature-state state="stable" >}}; and for kms provider, {{< feature-state feature-state state="beta" >}}?

Reply:
I'd explicitly list kms provider as beta since (whatever version it became beta in).

Reply:
updated but I definitely have the format wrong. guidance appreciated on these feature state flags